Wikipedia/Code/SearchLanguagesBarViewController.swift

```diff
@@ -0,0 +1,145 @@
+import Foundation
+
+@objc protocol SearchLanguagesBarViewControllerDelegate: class {
+    func searchLanguagesBarViewController(controller: SearchLanguagesBarViewController, didChangeCurrentlySelectedSearchLanguage language: MWKLanguageLink)
+}
+
+class SearchLanguagesBarViewController: UIViewController, WMFPreferredLanguagesViewControllerDelegate {
```

Comment:
@coreyfloyd seems like what we should be doing is dropping the prefix and using @objc(WMFClassNameHere).

`@objc (WMFSearchLanguagesBarViewController) class SearchLanguagesBarViewController {`

Anything exposed to or used by Objective-C should have a prefix.